src/netex_idf/stops.rs

```diff
@@ -236,15 +282,15 @@ fn load_stops(elem: &Element) -> Result<(CollectionWithId<StopArea>, CollectionW
     )?;
     add_stop_area_codes(&mut stop_areas, map_lda_zde);
 
-    let stop_points = load_stop_points(
+    let (stop_points, equipments) = load_stop_points(
         member_children.iter().filter(|e| e.name() == "Quay"),
         &mut stop_areas,
         &mut map_quay_lda,
     )?;
 
     update_stop_area_coords(&mut stop_areas, &stop_points);
 
-    Ok((stop_areas, stop_points))
+    Ok((stop_areas, stop_points, equipments))
```

Comment:
should I mutate collections instead of returning N CollectionWithId?

Reply:
I'm always struggling with this. I'm much more in favor in stateless functions in general but in most of our connectors, it really helps the readibility and make the code simpler to just send the `Collections` as a mutable reference.

Reply:
ok we keep like that